.github/workflows/prover-daily-test.yaml

```diff
@@ -0,0 +1,44 @@
+name: "Prover Daily Test"
+on:
+  # Allow us to manually run this specific workflow without a PR
+  workflow_dispatch:
+  # Until enabled on all PRs, run continuously at the beginning of each hour
+  schedule:
```

Comment:
you should be able to combine `prover-daily-test.yaml` and `prover-test.yaml` into a single workflow definition actually. that way you don't have to copy paste between two files.

```
on:
  workflow_dispatch:
  schedule:
    <your schedule>
  pull_request:
    <...>
```

and then the condition would be:

```
if: |
      contains(github.event.pull_request.labels.*.name, 'CICD:run-prover-test') || contains(fromJSON('["schedule", "pull_request"]'), github.event_name)
```

reference: https://docs.github.com/en/actions/learn-github-actions/expressions#example-matching-an-array-of-strings

Reply:
Great to know! However, two workflows run different jobs. The `prover-daily-test` runs while enabling the certain feature `MVP_TEST_INCONSISTENCY=1`. It is more time-consuming, so we want to run it only once a day.

Reply:
gotcha. You could still technically reuse the workflows and pass in an additional environment variable to the `prover-daily-test` though. 

I'll stamp to unblock so we can get something going though